libs/utils/src/lib/full-screen-mode/full-screen-mode.service.ts

```diff
@@ -1,18 +1,75 @@
-import { Injectable } from '@angular/core';
+import { Injectable, Inject } from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+
+interface FullscreenAPIMap {
+  requestFullscreen: string;
+  exitFullscreen: string;
+  fullscreenElement: string;
+}
+
+// Fullscreen API fallback map
+const FN_MAP: { [P in keyof FullscreenAPIMap]: string[] } = {
```

Comment:
Is there like a small library for this we could use? Or maybe create a small lib if that does not exist? Seems useful and your code seems pretty generic

Reply:
@NothingEverHappens We could replace this whole service with this: https://www.npmjs.com/package/angular-bigscreen